interfaces/builtin/dbus.go

```diff
@@ -213,10 +213,10 @@ func (iface *dbusInterface) StaticInfo() interfaces.StaticInfo {
 }
 
 // Obtain yaml-specified bus well-known name
-func (iface *dbusInterface) getAttribs(attribs map[string]interface{}) (string, string, error) {
+func (iface *dbusInterface) getAttribs(attribs interfaces.AttrGetter) (string, string, error) {
```

Comment:
This is a really bad method. Neither its name, nor its results, nor its documentation, say what it does.

Again, you're not making the situation any worse, so not for this PR, but we need to fix it.